lib/components/icon/customIconAvatar.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+import { ReactComponent as UserAvatar } from '../../assets/user_avatar.svg';
+
+const CustomIconAvatar = ({ className }) => (
+  <UserAvatar className={cn('custom-avatar', { [className]: className })} />
```

Comment:
classNames('foo', 'bar'); // => 'foo bar' not sure that object needed here

Reply:
oh yes. that makes sense.